spec.html

```diff
@@ -34587,15 +34587,15 @@ <h2>Syntax</h2>
           NonEmptyClassString
 
         NonEmptyClassString ::
-          ClassCharacter NonEmptyClassString?
+          ClassSetCharacter NonEmptyClassString?
 
-        ClassCharacter ::
-          [lookahead &notin; ClassReservedDouble] SourceCharacter but not ClassSyntaxCharacter
+        ClassSetCharacter ::
+          [lookahead &notin; ClassSetReservedOperator] SourceCharacter but not ClassSetSyntaxCharacter
           `\` CharacterEscape[+UnicodeMode]
-          `\` ClassAllowEscaped
+          `\` ClassSetReservedPunctuator
           `\` `b`
 
-        ClassReservedDouble :: one of
+        ClassSetReservedOperator :: one of
```

Comment:
This one seems confusing if its name doesn't contain something like "double".

Reply:
"Double" is potentially confusing because it often refers to IEEE 754 binary64 (cf. [Number value](https://tc39.es/ecma262/multipage/overview.html#sec-terms-and-definitions-number-value)), and the best way to prevent that confusion (other than by avoiding use of "double") is by following it with the noun that is being doubled. So I wouldn't object to `ClassSetReservedDoubleOperator`, but would nonetheless prefer something like `ClassSetReservedDoublePunctuator`.

Reply:
Alright, let’s go with `ClassSetReservedDoublePunctuator` then. Thanks!